doc/en/announce/sprint2016.rst

```diff
@@ -0,0 +1,80 @@
+
+python testing sprint June 20th-26th 2016
+======================================================
+
+.. image:: ../img/freiburg2.jpg
+   :width: 400
+
+The pytest core group is heading towards the biggest sprint
+in its history, to take place in the black forest town Freiburg
+in Germany.  As of February 2016 we have started a `funding
+campaign on Indiegogo to cover expenses
+<http://igg.me/at/pytest-sprint/x/4034848>`_ The page also mentions
+some preliminary topics.
+
+Participants
+--------------
+
+Here are preliminary participants who said they are likely to come,
+given some expenses funding::
+
+    Andreas Pelme, Sweden
+    Brianna Laugher, Australia
+    Bruno Oliveira, Brazil
+    Floris Bruhin, Switzerland
+    Florian Bruynooghe, UK
```

Comment:
before someone merges this I'm going to point out this is wrong here again :laughing: